examples/streetlights-operation-security.yml

```diff
@@ -0,0 +1,202 @@
+asyncapi: '2.4.0'
+info:
+  title: Streetlights Kafka API
+  version: '1.0.0'
+  description: |
+    The Smartylighting Streetlights API allows you to remotely manage the city lights.
+
+    ### Check out its awesome features:
+
+    * Turn a specific streetlight on/off 🌃
+    * Dim a specific streetlight 😎
+    * Receive real-time information about environmental lighting conditions 📈
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0
+
+servers:
+  test:
+    url: test.mykafkacluster.org:8092
+    protocol: kafka-secure
+    description: Test broker
+    security:
+      - saslScram: []
+  test_oauth:
+    url: test.mykafkacluster.org:8093
+    protocol: kafka-secure
+    description: Test port for oauth 
+    security:
+      - streetlights_auth:
+         - streetlights:write
+         - streetlights:read
+    
+ 
+defaultContentType: application/json
+
+channels:
+  smartylighting.streetlights.1.0.event.{streetlightId}.lighting.measured:
+    description: The topic on which measured values may be produced and consumed.
+    parameters:
+      streetlightId:
+        $ref: '#/components/parameters/streetlightId'
+    publish:
+      summary: Inform about environmental lighting conditions of a particular streetlight.
+      operationId: receiveLightMeasurement
+      traits:
+        - $ref: '#/components/operationTraits/kafka'
+      message:
+        $ref: '#/components/messages/lightMeasured'
+      security:
+       streetlights_auth:
+         - streetlights:write  
+  smartylighting.streetlights.1.0.action.{streetlightId}.turn.on:
+    parameters:
+      streetlightId:
+        $ref: '#/components/parameters/streetlightId'
+    subscribe:
+      operationId: turnOn
+      traits:
+        - $ref: '#/components/operationTraits/kafka'
+      message:
+        $ref: '#/components/messages/turnOnOff'
+      security:
+      # This operation level security implies the ability to send a message to 
+      # `smartylighting.streetlights.1.0.action.{streetlightId}.turn.on` with Authorization headers 
+      # with `streetlights:read` scope. It is also possible for the same channel when using `test_auth` 
+      # server to instead use credentials for security (scramSha256 in this example) specified on the server level
```

Comment:
Now that I'm writing the release notes, this comment seems confusing to me.

The spec says: 

> In cases where Server Security also applies, it MUST also be satisfied.

However, by reading this comment, I understand that only one (the server or rather the operation) security requirement should be satisfied.

Would you mind clarifying this @sekharbans-ebay? Also pinging Code Owners @derberg @fmvilas @dalelane

Reply:
> Now that I'm writing the release notes, this comment seems confusing to me.
> 
> The spec says:
> 
> > In cases where Server Security also applies, it MUST also be satisfied.
> 
> However, by reading this comment, I understand that only one (the server or rather the operation) security requirement should be satisfied.
> 
> Would you mind clarifying this @sekharbans-ebay? Also pinging Code Owners @derberg @fmvilas @dalelane

The example text is poorly worded - i ll reword this. My intention was to state that you could still  operate on the channel using security credentials supported at the server level perhaps thru a different operation.

Reply:
Would you mind creating a PR against `2022-04-release` branch with such a fix @sekharbans-ebay ?

Reply:
> Would you mind creating a PR against `2022-04-release` branch with such a fix (and the others I mentioned in other comments) @sekharbans-ebay ?

Done. Please review:
https://github.com/asyncapi/spec/pull/768